ray-operator/config/samples/tpu/tpu_list_devices.py

```diff
@@ -1,12 +1,21 @@
+import os
 import ray
 import jax
+import time
+
+from jax.experimental import multihost_utils
 
 ray.init()
 
 @ray.remote(resources={"TPU": 4})
 def tpu_cores():
-    return "TPU cores:" + str(jax.device_count())
+    cores = "TPU cores:" + str(jax.device_count())
+    print("TPU Worker: " + os.environ.get("TPU_WORKER_ID"))
+    time.sleep(10)
```

Comment:
why `time.sleep(10)`?

Reply:
I was testing different methods to ensure each Ray worker runs the Jax code once for each host (otherwise initialization times out), the preferred method is using `multihost_utils.sync_global_devices` so I'll update the code to reflect that.